packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -58,15 +58,23 @@
             },
 
             toggleAllCheckboxes: function () {
+                updatedStates = []
```

Comment:
We can probably call this `updatedState`

Reply:
done